deltachat-ios/Controller/ChatListController.swift

```diff
@@ -703,6 +704,11 @@ class ChatListController: UITableViewController {
                 self.handleEmptyStateLabel()
             }
         }
+        updateMarkArchivedReadButton()
```

Comment:
not sure if that is needed here, and a good place, did not test. if, it should be in an `if isArchive` branch

Reply:
it needs to be updated if new messages come in. I also just tested it it is triggered.